libethereum/ClientTest.cpp

```diff
@@ -101,8 +102,17 @@ void ClientTest::modifyTimestamp(int64_t _timestamp)
 
 void ClientTest::mineBlocks(unsigned _count)
 {
+    std::promise<void> allBlocksImported;
```

Comment:
This whole method will not work quite correctly in case some blocks are already being imported or mined, ideally we should check if mining is already started and just fail in this case, but this is not critical, we can leave it like this.